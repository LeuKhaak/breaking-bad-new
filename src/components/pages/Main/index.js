import React from "react";
import { Link } from "react-router-dom";
import * as styles from "./styles.module.scss";

function Main() {
  return (
    <section>
      <Link to="/catalog" className={styles.link}>
        MAIN. Go to catalog
      </Link>
    </section>
  );
}

export default Main;
