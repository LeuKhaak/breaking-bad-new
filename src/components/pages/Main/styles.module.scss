@import "../../../assets/styles/styles.scss";

.wrapper {
  font-size: 0;
  .image {
    width: 100%;
    height: calc(100vh - 10rem);
    object-fit: cover;
  }
}
