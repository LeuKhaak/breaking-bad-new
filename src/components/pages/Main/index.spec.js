import React from "react";
import { shallow } from "enzyme";
import Main from ".";

let component = null;
const setComponent = () => shallow(<Main />);
beforeEach(() => {
  component = setComponent();
});

describe("Main testing:", () => {
  test("is Main rendered", () => {
    component = setComponent({});
    expect(component).toMatchSnapshot();
  });
});
