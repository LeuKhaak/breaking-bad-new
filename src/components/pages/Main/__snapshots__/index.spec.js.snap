// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main testing: is Main rendered 1`] = `
<div>
  <t
    afterLoad={[Function]}
    alt="#"
    beforeLoad={[Function]}
    delayMethod="throttle"
    delayTime={300}
    effect=""
    placeholderSrc={null}
    src="test-file-stub"
    threshold={100}
    useIntersectionObserver={true}
    visibleByDefault={false}
    wrapperClassName=""
  />
</div>
`;
